js/client-battle-tooltips.js

```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
 
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
+				if (this.battle.gameType === 'doubles') {
+					if (move.target === 'allAdjacent') {
+							text += '<p class="movetag">&#x2713; Hits both foes and ally.</p>';
```

Comment:
Also would not use checkmark for this. Move targets aren't allies.

Consider `fa-dot-circle-o` or `fa-bullseye`?